src/iinit.cpp

```diff
@@ -123,7 +134,9 @@ int main( int argc, char **argv )
     }
 
     // Create ~/.irods/ if it does not exist
-    mkrodsdir();
+    if (mkrodsdir() != 0) {
+        std::exit(EXIT_FAILURE);
```

Comment:
Any reason to prefer this over just returning a code from `main`? Really asking.

Reply:
Think I did a `std::exit(...)` since I did that originally within `mkrodsdir(...)`... We can change that to a `return` instead.

Reply:
Yeah, seeing `return` in `main()` feels more intuitive.